### ISSUE TITLE
List - screenreader does not announce correct number of items in list

### ISSUE THREAD

**Issue by mikerodonnell89** [SAP/fundamental-ngx#303]
When using the ChromeVox screenreader, after focusing an `<fd-list>`, the screenreader says it's a list containing zero items when there are list items present.

Angular 6.0.0, fundamental-ngx 0.0.20-beta
